elephant/spike_train_correlation.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+"""
+Spike train correlation
+
+This modules provides functions to calculate correlations between spike trains.
+
+:copyright: Copyright 2014 by the Elephant team, see AUTHORS.txt.
+:license: Modified BSD, see LICENSE.txt for details.
+"""
+from __future__ import division
+import numpy as np
+
+
+def corrcoef(binned_sts, binary=False):
```

Comment:
What is the reason for providing a binned spike train object, rather than a list of `neo.SpikeTrain`s?